src/airflow/dags/ukb_ppp_eur.py

```diff
@@ -0,0 +1,42 @@
+"""Airflow DAG to ingest and harmonise UKB PPP (EUR) data."""
+
+from __future__ import annotations
+
+from pathlib import Path
+
+import common_airflow as common
+from airflow.models.dag import DAG
+
+CLUSTER_NAME = "otg-ukb-ppp-eur"
+
+# Input location.
+UKB_PPP_EUR_STUDY_INDEX = "gs://gentropy-tmp/batch/output/ukb_ppp_eur/study_index.tsv"
```

Comment:
Using two manual steps as described here: https://github.com/opentargets/issues/issues/3234#issuecomment-2180776005
* Step 1: Mirror from Synapse into our Google Cloud storage
* Step 2: Convert into a Spark-ingestible format using Google Batch

As we discussed with @addramir previously, it probably never makes sense to automate step 1 (too much effort, too little gain). As for step 2, it might make sense at some point to invest effort into automating it as an Airflow DAG, but probably not right away